lib/Index/IndexDatastore.cpp

```diff
@@ -763,17 +766,17 @@ void StoreUnitRepo::pollForUnitChangesAndWait() {
 
       auto I = knownUnits.find(unitName);
       if (I != knownUnits.end() && I->getValue() != modTime) {
-        events.push_back({IndexStore::UnitEvent::Kind::Modified, unitName.str()});
+        events.push_back({IndexStore::UnitEvent::Kind::Modified, unitName.str(), /*isInitialScan=*/false});
```

Comment:
Should we pass in `isInitialScan` to `pollForUnitChangesAndWait()`?  It is used for the initial scan when `listenToUnitEvents` is false.  Otherwise I think this should have a comment to explain why false would always be correct here.

Reply:
Good point, added as parameter in the APIs.